conf/messages

```diff
@@ -258,6 +263,31 @@ paye.ats.treasury_spending.para2=All figures are rounded to the nearest pound.
 paye.ats.treasury_spending.para3=The figures in the table above are intended as an illustration of how taxes are spent and not as a direct link between your Income Tax, National Insurance contributions and any specific expenditure.
 paye.ats.treasury_spending.para4=Spending information is published by HM Treasury.
 
+# View ATS - Summary (PAYE)
+#============================================================
+paye.ats.summary.html.title=Your income and taxes
+paye.ats.summary.title=Your income and taxes
+paye.ats.summary.lede=This is for your information only. You do not need to contact us as this is not a bill or demand for payment.
+paye.ats.summary.total_income.title=Taxable income
+paye.ats.summary.total_income.link=Your taxable income
+paye.ats.summary.total_income.description=This is your total taxable income for the year.
+paye.ats.summary.tax_free.title=Tax-free amount
+paye.ats.summary.tax_free.link=Your tax-free amount
+paye.ats.summary.tax_free.description=This is the amount you received without paying tax.
+paye.ats.summary.tax_and_nics.title=Income Tax and National Insurance
+paye.ats.summary.tax_and_nics.link=Your Income Tax and National Insurance contributions
+paye.ats.summary.tax_and_nics.description=This is {0}% of your taxable income. For every £1 of income, you paid {0}p in Income Tax and National Insurance contributions.
+paye.ats.summary.capital_gains.title=Capital Gains Tax
+paye.ats.summary.capital_gains.link=Your Capital Gains Tax
+paye.ats.summary.capital_gains.description=This is how much tax you pay on profits above your annual tax-free allowance when you sell, or give away, something that has increased in value.
+paye.ats.summary.total_tax.your_total_tax=Total tax
+paye.ats.summary.taxable_income.your_total_tax.msg_ternary=Your {0}, {1} and {2}.
+paye.ats.summary.taxable_income.your_total_tax.msg_binary=Your {0} and {1}.
+paye.ats.summary.taxable_income.your_total_tax.msg_unary=Your {0}.
+paye.ats.summary.taxable_income.your_total_tax.description_total_income_tax=total Income Tax
```

Comment:
Little t?

Reply:
@russellthorn Can you please address this?

Reply:
agreed
done

Reply:
Is little T correct and inline with the correct naming conventions we have been directed to use from the stakeholders?